lib/dal/src/func/binding.rs

```diff
@@ -144,6 +144,25 @@ impl AttributeFuncDestination {
         };
         Ok(schema_variant_id)
     }
+
+    pub async fn get_name_of_destination(&self, ctx: &DalContext) -> FuncBindingResult<String> {
+        let name = match self {
+            AttributeFuncDestination::Prop(prop_id) => Prop::get_by_id(ctx, *prop_id).await?.name,
+            AttributeFuncDestination::OutputSocket(output_socket_id) => {
+                OutputSocket::get_by_id(ctx, *output_socket_id)
+                    .await?
+                    .name()
+                    .to_string()
+            }
+            AttributeFuncDestination::InputSocket(input_socket_id) => {
```

Comment:
nit for the future: I like to use `Self` when possible for readability and suppleness for refactoring. So the first one would be `Self::Prop` instead of `AttributeFuncDestination::Prop`.